contributed/Using-Postgres-catalog.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accessing DC2 data in PostgreSQL at NERSC"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates use of the PostgreSQL database at NERSC.  Currently the only recommended dataset is the object catalog for Run1.2i.  Run1.2p will be available \n",
+    "shortly after the current (as of Jan. 18) reprocessing is complete.\n",
+    "\n",
+    "### Prerequisites\n",
+    "* A file ~/.pgpass containing a line like this:\n",
+    "\n",
+    "`nerscdb03.nersc.gov:54432:desc_dc2_drp:desc_dc2_drp_user:`_password_\n",
+    "\n",
+    "This line allows you to use the desc_dc2_drp_user account, which has *SELECT* privileges on the database, without entering a password in plain text. There is a separate account for adding to or modifying the database. .pgpass must be protected so that only owner may read and write it.\n",
+    " \n",
+    "You can obtain the file by running the script `/global/common/software/lsst/dbaccess/postgres_reader.sh`.  It will copy a suitable file to your home directory and set permissions.  \n",
+    "\n",
+    "If you already have a `.pgpass` file in your home directory the script will stop without doing anything to avoid clobbering your file.  In that case, see the file `reader.pgpass` in the same directory.  You can merge it into your `.pgpass` file by hand.  \n",
+    "\n",
+    "* Access to the psycopg2 package which provides a Python interface to PostgreSQL. If it's not already available, do a local pip install.  Then you may have to \n",
```

Comment:
Is it time to add psycopg2 to the NERSC shared python installation?

Reply:
I'm in favor.  It would lower the barrier to use of PostgreSQL significantly.

Reply:
Now psycopg2==2.7.5 is part of the desc-python kernel, so it's probably best to remove references to local installs and suggest the user use the desc-python kernel on juptyer-dev.